src/main/java/pl/coderstrust/accounting/logic/pdf/PdfSet.java

```diff
@@ -0,0 +1,118 @@
+
+package pl.coderstrust.accounting.logic.pdf;
+
+import org.apache.pdfbox.pdmodel.common.PDRectangle;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import pl.coderstrust.accounting.model.Invoice;
+import pl.coderstrust.accounting.model.InvoiceEntry;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+public class PdfSet {
```

Comment:
this class name is meaningless - please think about better name

Reply:
done